pkg/platform/openstack/loadbalancer.go

```diff
@@ -411,3 +411,35 @@ func listOpenStackOctaviaProviders(client *gophercloud.ServiceClient) ([]provide
 		return providersList, nil
 	}
 }
+
+// Iterate on pool members and check their address against provided list
+// addresses of current master/bootstrap nodes. Remove all surplus members,
+// which address doesn't exists on that list.
+func purgeOpenStackLbPoolMember(client *gophercloud.ServiceClient, poolId string, addresses []string) error {
+	page, err := pools.ListMembers(client, poolId, nil).AllPages()
+	if err != nil {
+		return errors.Wrap(err, "failed to get LB member list")
+	}
+
+	members, err := pools.ExtractMembers(page)
+	if err != nil {
+		return errors.Wrap(err, "failed to extract LB members list")
+	}
+
+	for _, member := range members {
+		found := false
+		for _, address := range addresses {
+			if address == member.Address {
+				found = true
+				break
```

Comment:
Instead of using `found` flag you could just use loops labels: mark loop in 429 as `members_loop` and make this a `continue members_loop`.